website/docs/d/pi_datacenter.html.markdown

```diff
@@ -38,6 +38,7 @@ Example usage:
 
 Review the argument references that you can specify for your data source.
 
+- `pi_cloud_instance_id` - (Required if private datacenter, String) The GUID of the service instance associated with an account.
```

Comment:
That first part can be in the comments, like we did it in the instance
https://github.com/powervs-ibm/terraform-provider-ibm/blob/7b568a12e5926b8557ace2b12a794818fe9a3beb/website/docs/r/pi_instance.html.markdown?plain=1#L69C1-L69C2

Reply:
Updated